04-Docker.md

```diff
@@ -1,62 +1,76 @@
 # Part IV: Docker
 
-## Objectives: 
-By the end of this module, the SRE will: 
-- Describe the directives from a sample dockerfile 
-- Explain layers and the docker cache 
-- Start a docker container that runs ubuntu linux 
+## Objectives:
+By the end of this module, the SRE will:
+- Describe the directives from a sample dockerfile
+- Explain layers and the docker cache
+- Start a docker container that runs ubuntu linux
 - Describe the following container concepts:
   - base image
   - single pid
-  - command vs entrypoint
+  - `CMD` vs `ENTRYPOINT`
   - logs
   - volumes
   - layers and ordering
   - caching
   - build
   - push
 
-## Deliverables 
-1. Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
+## Deliverables
+Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
 
-### Docker 
+### Docker
+  1. Setup
+  - https://www.youtube.com/watch?v=JprTjTViaEA
+  - https://jonnylangefeld.github.io/learning/Docker/How%2Bto%2BDocker.html
   - Install docker
-  - `docker login` to reactiveops quay and dockerhub
+  - login to reactiveops quay and dockerhub via the terminal
```

Comment:
s/reactiveops/fairwinds/g

Or maybe we remove the company name altogether? Open to discussion on that.

Reply:
I think I prefer the company name to be there. It implies that I am not supposed to make my own account...but I suppose if you must reach out to get access, that ALSO implies the same thing...I'm open to either.

Reply:
I also see what you are saying about moving away from Fairwinds specific language.

Reply:
Also @ejether is there a preference between Fairwinds or FairwindsOps?

Reply:
Fairwinds, please.

I think the conceptual shift is the target audience. Since this is open source and @kimschles  has been using is as a resource externally, using FWOPs specific language _may_ diminish that? but at the same time, it complicates it for us, as you pointed out. 

I'm fine with keeping in internal specific so that it is most useful for us and treating the OS aspects as incidental. Do you have any opinions @kimschles ?

Reply:
I think our priorities should be in this order: 
1. A curriculum that works for apprentice-level engineers at FairWinds
2. Making things accessible to folks from outside the org

So, what I mean is let's keep the FWOPs language for now. :)

Reply:
That being said, I think changing 'pod leader' to 'mentor' was a good call.